resources/kernel/configs.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+
+# Copyright 2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0 OR BSD-3-Clause
+
+# This scripts determines the busybox version to be used based on the
+# current glibc version (1.32.1 for glibc version > 2.31)
+# Also kernel version and config parameters are exported from here.
+
+KERNEL_VERSION="4.14.176"
+KERNEL_CFG="microvm-kernel-initramfs-hello-x86_64.config"
+
+
+cat > glibc_ver.c << EOF
+#include <stdio.h>
+#include <gnu/libc-version.h>
+int main (void) { puts (gnu_get_libc_version ()); return 0; }
+
+EOF
+
+GLIBC_VER=`gcc -o glibc_ver glibc_ver.c 2>/dev/null && ./glibc_ver 2>/dev/null`
+
+#Delete the files generated
+rm -f glibc_ver* 2> /dev/null
+
+if [[ -z $GLIBC_VER ]]; then
+	echo "Unable to determine glibc version, using default Busybox: 1.26."
+	BUSYBOX_CFG="busybox_1_26_static_config"
+	BUSYBOX_VERSION="1.26.0"
+else
```

Comment:
This part can be omitted for brevity; if `$GLIBC_VER` is not set, `[[ $GLIBC_VER < 2.31 ]]` evaluates to `true`

Reply:
This part I kept separate because, if this gets logged through redirect (`> 1 2>&1 `), it will help in debugging why a particular version was selected, so this becomes explicit, but without that it's implicit.

Reply:
Also, if in future one has to make `1.32` as a default (likely as more and more distros move to `glibc 2.31+`), relying on the above evaluation to `true` won't help. So I am not entirely sure it's a good idea to omit this part.